ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml

```diff
@@ -0,0 +1,52 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile
+  to: kubevirt-template-validator
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
```

Comment:
`tag_specification` is legacy and should not be used except for configs with legacy tests. 

Please use `releases` instead: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release